terraform/testcases/otlp_mock/parameters.tfvars

```diff
@@ -1,2 +1,5 @@
 # data type will be emitted. Possible values: metric or trace
 soaking_data_mode = "trace"
+
+#sample app image
+sample_app_image = "public.ecr.aws/aws-otel-test/aws-otel-java-spark:latest"
```

Comment:
You'll want to change it in more places than this one. There are a lot of testcases (`logzio_exporter_trace_mock`, `newrelic_exporter_metric_mock`, etc.) that use the default sample app, which is the spark app.